Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.py

```diff
@@ -1983,6 +1997,43 @@ def get_user_data(user_response):
     return user_data
 
 
+def offboard_machine_command(client: MsClient, args: dict):
+    """Offboard machine from defender.
+
+    Returns:
+        (str, dict, dict). Human readable, context, raw response
+    """
+    headers = ['ID', 'Type', 'Requestor', 'RequestorComment', 'Status', 'MachineID', 'ComputerDNSName']
```

Comment:
Please add a validate if the arguments were indeed provided

```suggestion
    if not args.get('machine_id') or not args.get('comment'):
        raise ValueError("something")
    headers = ['ID', 'Type', 'Requestor', 'RequestorComment', 'Status', 'MachineID', 'ComputerDNSName']
```